include/n64dd.h

```diff
@@ -40,10 +45,10 @@ typedef struct n64ddStruct_80121220 {
     s32 (*unk_20)(struct MapData*);
     s32 (*unk_24)(void);
     s32 (*unk_28)(struct PlayState*);
-    s32 (*unk_2C)(MapMarkData***);
-    s32 (*unk_30)(MapMarkData***);
-    void (*unk_34)(PauseMapMarksData**);
-    void (*unk_38)(PauseMapMarksData**);
+    s32 (*unk_2C)(struct MapMarkIconData***[3]);
+    s32 (*unk_30)(struct MapMarkIconData***[3]);
+    void (*unk_34)(struct PauseMapMarkData**[3]);
+    void (*unk_38)(struct PauseMapMarkData**[3]);
```

Comment:
I find these and the `void(**)(struct PlayState*)` below much less readable than with the typedef. Is it really worth it?